client/src/containers/LoginContainer.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import dotenv from 'dotenv';
+import styled from 'styled-components';
+import Login from '../components/Login';
+import { API_SERVER } from '../utils/constants';
+import { RootState } from '../modules';
+import { login } from '../modules/login';
+
+dotenv.config();
```

Comment:
login.ts 파일에서 process.env를 사용하시는데 문제가 있을 것 같습니다.

```js
import 'dotenv/config';
```

[여기](https://github.com/connect-foundation/2019-18/pull/104/files#diff-fdb977aad5f420b65f8d4dfe84af55c8R4)에 위 구문을 한 번만 사용하시고 다른부분의 코드는 지우셔도 될것 같습니다.
추가로 아래 링크를 공유드리니 참고해보세요.

> 1. Preload dotenv: node --require dotenv/config index.js (Note: you do not need to import dotenv with this approach)
> 2. Import dotenv/config instead of dotenv (Note: you do not need to call dotenv.config() and must pass options via the command line or environment variables with this approach)
> 3. Create a separate file that will execute config first as outlined in this comment on #133
> https://www.npmjs.com/package/dotenv#how-do-i-use-dotenv-with-import-